backend/src/main/java/com/bakdata/conquery/models/config/StorageConfig.java

```diff
@@ -21,4 +23,14 @@
 	private boolean useWeakDictionaryCaching = true;
 	@NotNull
 	private Duration weakCacheDuration = Duration.hours(48);
+	
+	/**
+	 * Flag for the {@link SerializingStore} whether to delete values from the underlying store, that cannot be mapped to an object anymore.
+	 */
+	private boolean removeUnreadablesFromStore = false;
+	
+	/**
+	 * When set, all values that could not be deserialized from the persistent store, are dump into individual files.
+	 */
+	private Optional<File> unreadbleDataDumpDirectory = Optional.empty();
```

Comment:
reicht null nicht?

Reply:
Ich fand den guava anspruch ganz gut, dass man kein null ausgibt, damit man sich darüber Gedanken macht wenn mal etwas nicht gesetzt ist.
In einer Klasse intern kann man weiterhin null nutzen, aber da Konfigurationen genau dazu ausgelegt sind, dass sie andere Sachen konfigurieren finde ich optional hier besser